crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -280,18 +285,11 @@ fn handle_own_line_comment_around_body<'a>(
     }
 
     // Check if we're between bodies and should attach to the following body.
-    handle_own_line_comment_between_branches(comment, preceding, locator)
-        .or_else(|comment| {
-            // Otherwise, there's no following branch or the indentation is too deep, so attach to the
-            // recursively last statement in the preceding body with the matching indentation.
-            handle_own_line_comment_after_branch(comment, preceding, locator)
-        })
-        .or_else(|comment| {
-            // If the following node is the first in its body, and there's a non-trivia token between the
-            // comment and the following node (like a parenthesis), then it means the comment is trailing
-            // the preceding node, not leading the following one.
-            handle_own_line_comment_in_clause(comment, preceding, locator)
```

Comment:
This is no longer needed, and has been removed. It's handled as part of `handle_parenthesized_comment`.